### ISSUE TITLE
Decide on clear behaviour for a bang with empty query

### ISSUE THREAD

**Issue by psidex** [psidex/CustomBangSearch#30]
For example, if `!m new york` takes you to `https://www.google.com/maps/search/New+York`, should `!m` on it's own take you to `https://www.google.com/maps/`? Or `https://www.google.com/`? or `https://www.google.com/maps/search/`?

Maybe the behaviour should be customisable in the options page? per bang?

**Comment by Jorengarenar** [comment]
I think the default behaviour should be kept consistent with DDG (i.e. empty bang redirects to base domain page).
Extension could as an added bonus provide optional field where users can specify fallback URL.

**Comment by cttps** [comment]
I agree that default should stay consistent with what DDG does. But customization per bang would be very useful.

**Comment by psidex** [comment]
v0.11.0 introduces default URL options
